app/config/FrontendAppConfig.scala

```diff
@@ -22,83 +22,95 @@ import com.google.inject.{Inject, Singleton}
 import controllers.routes
 import models.VatEnrolment
 import play.api.i18n.Lang
-import play.api.mvc.Request
+import play.api.mvc.{Call, Request}
 import play.api.{Configuration, Environment}
-import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import uk.gov.hmrc.play.language.LanguageUtils
 import utils.PortalUrlBuilder
 
 @Singleton
-class FrontendAppConfig @Inject() (override val runModeConfiguration: Configuration, environment: Environment) extends ServicesConfig with PortalUrlBuilder {
+class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration,
+                                  environment: Environment,
+                                  servicesConfig: ServicesConfig,
+                                  override val languageUtils: LanguageUtils) extends PortalUrlBuilder {
 
-  override protected def mode = environment.mode
+  import servicesConfig._
```

Comment:
can this live with the other imports?

Reply:
no, cos it's importing the methods from within the servicesConfig instance